config/global.ini.php

```diff
@@ -104,6 +104,14 @@
 ; if configured to log in a file, log entries will be made to this file
 logger_file_path = tmp/logs/piwik.log
 
+; if set to 1, will enable monolog's FingersCrossedHandler, which buffers all logs and flushes them if a warning or error
+; log is seen. so the logs will contain debug logs, but only for requests where an error occurred. does not support the screen log writer
+enable_fingers_crossed_handler = 0
+
+; if enable_fingers_crossed_handler is set to 1, and this is set to 1, then the FingersCrossedHandler will stop buffering
+; on the first warning/error. if another one occurs later in the request, there will be no extra debug logs for them.
+fingers_crossed_stop_buffering_on_activation = 0
```

Comment:
FingersCrossedHandler has an option where it stops buffering after the first error. I assume it's to avoid buffering too many logs in case there are lots of errors? I'm not exactly sure, but figured someone might want to set it true (by default I've left it disabled so every error triggers debug logs to be flushed).

Reply:
Same here I reckon we could keep the setting and see how it behaves in production and then possibly remove the setting and always have it disabled but to be seen after some more testing